lib/bolt/node/orch.rb

```diff
@@ -91,6 +91,7 @@ def unwrap_bolt_result(result)
         return result
       end
 
+      # Is there a reason the result isn't logged here like with other transports?
```

Comment:
Its retrievable from orchestrator. We could log the job-id for everything that goes through _run_task